### PR TITLE
enh(epp): applications-ipfabric-api

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [X] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md [hunk 1]
```diff
@@ -5,41 +5,54 @@ title: IP Fabric API
...
-
-## Pack Assets
+## Pack assets
 
 ### Templates
 
-The Centreon Monitoring Connector **IP Fabric API** brings a host template:
+The Monitoring Connector **IP Fabric API** brings a host template:
+
+* **App-Ipfabric-Api**
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md:14]:
```suggestion
* **App-Ipfabric-Api-custom**
```

File: pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md [hunk 2]
```diff
@@ -5,41 +5,54 @@ title: IP Fabric API
...
 
-* App-Ipfabric-Api-custom
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-Ipfabric-Api" label="App-Ipfabric-Api">
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md:19]:
```suggestion
<TabItem value="App-Ipfabric-Api-custom" label="App-Ipfabric-Api-custom">
```

File: pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md [hunk 3]
```diff
@@ -5,41 +5,54 @@ title: IP Fabric API
...
 
-It brings the following service template:
+| Service Alias     | Service Template                   | Service Description |
+|:------------------|:-----------------------------------|:--------------------|
+| Path-Verification | App-Ipfabric-Path-Verification-Api | Check paths status |
 
-| Service Alias     | Service Template                   | Service Description | Default |
-|:------------------|:-----------------------------------|:--------------------|:--------|
-| Path-Verification | App-Ipfabric-Path-Verification-Api | Check paths' status | X       |
+> The services listed above are created automatically when the **App-Ipfabric-Api** host template is used.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md:25]:
```suggestion
> The services listed above are created automatically when the **App-Ipfabric-Api-custom** host template is used.
```

File: pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md [hunk 4]
```diff
@@ -48,114 +61,297 @@ The IP Fabric Monitoring Connector includes a Host discovery provider to automat
 
 Ensure you have a IP Fabric API Key with required privileges and IP Fabric's API address.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md:62]:
```suggestion
Ensure you have an IP Fabric API Key with required privileges and IP Fabric's API address.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-ipfabric-api.md [hunk 5]
```diff
@@ -5,41 +5,55 @@ title: IP Fabric API
...
-
-## Contenu du Pack
+## Contenu du pack
 
 ### Modèles
 
-Le connecteur de supervision Centreon **IP Fabric API** apporte un modèle d'hôte :
+Le connecteur de supervision **IP Fabric API** apporte un modèle d'hôte :
+
+* **App-Ipfabric-Api**
```
[the remark this answers is not in the document]
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-ipfabric-api.md]: Please accept the commit suggestions @garnier-quentin (Ivan's script will soon be updated with this enhancement).

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**